app.js

```diff
@@ -19,6 +19,20 @@ app.use('/static', express.static('public'));
 
 app.get('/', (req, res) => res.render('index'));
 
+app.get('/dashboard/:id/:page?', function(req, res) {
+  var pageParam = req.params.page;
+  var pageName = 'dashboardView';
+  if (pageParam === 'edit') pageName = 'dashboardEdit';
+
+  res.render(
+    pageName,
+    {
+      layout: 'dashboard',
```

Comment:
Is layout being used for anything?

Reply:
This is how handlebars knows to use the dashboard layout (which has a custom header and side nav) instead of the default index page.